### ISSUE TITLE
Add alt text to featured images

### ISSUE THREAD

**Issue by jclwilson** [jclwilson/jclwilson.github.io#6]
Alt text has been omitted from the design, now that tags are included to allow for alt text old posts should have alt text added via front matter using `image-alt` variables.

Alt text should be descriptive of the content of the image, it should not be identical to the caption, which is reserved for title, creator, date, and ownership.
